meshtastic/interfaces/ble/management_service.py

```
@@ -0,0 +1,369 @@
+                        iface._validate_management_preconditions()
+                        refreshed_existing_client = (
+                            iface._get_management_client_if_available(address)
+                        )
+                        if refreshed_existing_client is None:
+                            raise iface.BLEError(ERROR_MANAGEMENT_TARGET_CHANGED)
+                return command(refreshed_existing_client)
+
+            if target_address is None:
+                raise iface.BLEError(ERROR_MANAGEMENT_ADDRESS_REQUIRED)
+            with iface._management_target_gate(target_address):
+                with iface._connect_lock:
```

Ruff (0.15.5)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)